dapp-manager/src/main/kotlin/jp/co/soramitsu/dapp/manager/config/DappManagerConfig.kt

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright D3 Ledger, Inc. All Rights Reserved.
+ *  SPDX-License-Identifier: Apache-2.0
+ */
+
+package jp.co.soramitsu.dapp.manager.config
+
+const val DAPP_MANAGER_NAME = "dapp-manager"
+
+const val DAPP_DOMAIN = "dapp"
+
+interface DappManagerConfig {
+
+    val accountId: String
+
+    val registratorAccountId: String
+
+    val repository: String
+
+    val journal: String
+
+    val dappAccountsStorage: String
+
+    val brvsAccountId: String
+
+    val pubKeyPath: String
```

Comment:
According to the list of improvments made by DevOps team, we must store key pair content directly. No paths.

Reply:
Done